resources/bric/ColorPicker_es.properties

```diff
@@ -8,15 +8,16 @@
 # accordance with the license agreement you entered into with
 # Jeremy Wood. For details see accompanying license terms.
 #
-ColorPickerDialogTitle=Elegir Color

```

Comment:
This file mixes Unix- (LF) and Windows-style (CRLF) end of line marks as can be seen by the spurious `^M` at the end of lines. Please, remove them.

Reply:
That's true. I didn't know where those symbols were coming from, but left them because they were in the original file.